pingdom/pingdom.go

```diff
@@ -70,36 +58,18 @@ func NewClientWithConfig(config ClientConfig) (*Client, error) {
 	c.Checks = &CheckService{client: c}
 	c.Maintenances = &MaintenanceService{client: c}
 	c.Probes = &ProbeService{client: c}
-	c.Teams = &TeamService{client: c}
-	c.PublicReport = &PublicReportService{client: c}
-	c.Users = &UserService{client: c}
 	return c, nil
 }
 
 // NewClient returns a Pingdom client with a default base URL and default HTTP client.
 // Deprecated: Use NewClientWithConfig
-func NewClient(user string, password string, key string) *Client {
-	config := ClientConfig{
-		User:     user,
-		Password: password,
-		APIKey:   key,
-	}
-	c, _ := NewClientWithConfig(config)
-	return c
-}
-
-// NewMultiUserClient extends NewClient to allow Multi-User authentication.
-// Deprecated: Use NewClientWithConfig
-func NewMultiUserClient(user string, password string, key string, accountEmail string) *Client {
-	config := ClientConfig{
-		User:         user,
-		Password:     password,
-		APIKey:       key,
-		AccountEmail: accountEmail,
-	}
-	c, _ := NewClientWithConfig(config)
-	return c
-}
+// func NewClient(key string) *Client {
```

Comment:
nit: what do you think about removing the commented out code here and just exclusively use `NewClientWithConfig`?

Reply:
I can do that